From 9635f50c0d4f9947a62ba065abeae350bbbb2c21 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Thu, 19 Dec 2013 11:19:10 +0100
Subject: [PATCH 10/10] virt-image: remove "qcow" from the manual

The "qcow" format is not supported by qemu-kvm

RHEL-ONLY

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1024180

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
---
 man/en/virt-image-xml.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/en/virt-image-xml.pod b/man/en/virt-image-xml.pod
index 2f3ea39..3ebea7f 100644
--- a/man/en/virt-image-xml.pod
+++ b/man/en/virt-image-xml.pod
@@ -178,7 +178,7 @@ the I<size> attribute giving the size of the disk in MB.
 =item *
 
 the I<format> attribute giving the format of the disk file. Currently, this
-can be one of: C<raw> C<iso>, C<qcow>, C<qcow2>, or C<vmdk>.
+can be one of: C<raw>, C<iso>, C<qcow2>, or C<vmdk>.
 
 =back
 
-- 
1.8.5.3

